src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IAnonymousFunctionExpression.cs

```diff
@@ -409,15 +410,15 @@ static void F()
             var lambdaSyntax = (LambdaExpressionSyntax)variableDeclaration.Declaration.Variables.Single().Initializer.Value;
 
             var variableDeclarationGroupOperation = (IVariableDeclarationGroupOperation)semanticModel.GetOperation(variableDeclaration);
-            var variableTreeLambdaOperation = (IAnonymousFunctionOperation)variableDeclarationGroupOperation.Declarations.Single().Declarators.Single().Initializer.Value;
+            var variableTreeLambdaOperation = ((IDelegateCreationOperation)variableDeclarationGroupOperation.Declarations.Single().Declarators.Single().Initializer.Value).Target;
```

Comment:
>IDelegateCreationOperation [](http://example.com/codeflow?start=48&length=26)

I don't think this is keeping the test correct. Please break the test in a different fashion, such as removing the `F()` definition so the body is still unbound.

Reply:
Will add to the next PR, thanks.